siteapp/views.py

```diff
@@ -25,30 +25,90 @@ def homepage(request):
         "login_form": LoginForm,
     })
 
-def folder_list(request):
-    # Get the folders the user can see. Folders are accessible to a user
-    # just when they can see a Project within it or if they
-    # are an admin of the folder, so we go backwards from Projects
-    projects = Project.get_projects_with_read_priv(request.user, request.organization)
-    folders = (Folder.objects.filter(projects__in=projects)
-            | Folder.objects.filter(organization=request.organization, admin_users=request.user))\
-          .distinct()
 
-    # Count up the total number of projects in each folder. For admins, so they
-    # can know why they can't delete a folder.
-    from django.db.models import Count
-    folders = folders.annotate(project_count=Count('projects'))
+def assign_project_lifecycle_stage(projects):
+    # Define lifecycle stages.
+    # Because we alter this data structure in project_list,
+    # we need a new instance of it on every page load.
+    lifecycle_stages = [
```

Comment:
Seems like we are sticking configuration information inside the source code. Shouldn't we be able to define different life cycles for different users/orgs?

Reply:
Yes.

Reply:
So we can treat this as temporary technical debt?

Reply:
I think that's best. It would be easy to move this configuration somewhere else later, e.g. into the database and make it per-organization or something. I wouldn't call this tech debt - we're not making things harder later by leaving it here, we just haven't done the additional logic of getting this data from somewhere else yet.